www/js/theme_editor.js

```diff
@@ -1,36 +1,16 @@
 const h = require('hyperscript');
-// const fs = require('fs');
 const Noty = require('noty');
 const themes = require('./themes.json');
 
 const { store, analytics } = require('electron').remote.require('./app');
 
-// const imagesPath = 'assets/custom_backgrounds';
-
 const defaultTheme = themes[0];
 
-/* defaultTheme.bgImage = '';
-const getCurrentTheme = () => {
-  const currentThemeString = localStorage.getItem('customTheme');
-  if (currentThemeString) {
-    try {
-      return JSON.parse(currentThemeString);
-    } catch (error) {
-      new Noty({
-        type: 'error',
-        text: `There is an error reading current theme.
-        Try checking theme file for errors. If error persists,
-        report it at www.sttm.co`,
-        timeout: 3000,
-        modal: true,
-      }).show();
-      return defaultTheme;
-    }
-  }
-  return defaultTheme;
-}; */
+const themesWithCustomBg = themes
+                            .filter(theme => theme.type === 'COLOR' || theme.type === 'SPECIAL')
+                            .map(theme => theme.key);
 
-const swatchFactory = themeInstance =>
+const swatchFactory = (themeInstance, isItCustom) =>
```

Comment:
Convention is usually to omit the `It` and just have `isCustom`
Also, it might be best to assign a default value of `false`